sdk/websites/Azure.ResourceManager.AppService/src/autorest.md

```diff
@@ -849,4 +849,64 @@ directive:
             "Aborted",
             "Running"
         ]
+  - from: CommonDefinitions.json
+    where: $.definitions
+    transform: >
+        $["HybridConnectionCollection"] = {
+            "description": "Collection of hostname bindings.",
+                  "required": [
```

Comment:
fix the indentation

Besides, shall we have to re-write so many properties? We can use `$["HybridConnectionCollection"]["xxxx"]` to reduce the impact

Reply:
Ok， I will reference the 'HybridConnectionCollection' definition from AppServicePlans.json instead.